{% extends 'base.html' %} {% block content %}
{% load static %}
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="{% static 'css/estil.css' %}">
    <title>UOH invest center</title>
    <html lang="en">
        

    <div class="content">
        <h1>Software Di Genoma Lab</h1>
        <h2>Last updated on May 7, 2023</h2>
        <div>
            <h1>DiGenoma Lab software</h1>
            <p>At DiGenoma Lab, we have developed a suite of open-source software tools to facilitate high-throughput genomics research. Our software is designed with reproducibility in mind, and we use containerization technologies like Docker and Singularity to ensure that our pipelines run consistently across different environments and platforms.</p>
    
            <p>Our software is optimized to run on HPC clusters, and we take full advantage of high-performance computing resources to enable rapid analysis of large-scale genomic datasets.</p>
    
            <p>For example, our genome assembly software includes <a href="https://github.com/adigenova/wengan" target="_blank">Wengan</a>, an ultra-fast and accurate hybrid assembler that can leverage long- and short-read data for optimal genome reconstruction. We also offer <a href="https://github.com/adigenova/Fast-SG" target="_blank">Fast-SG</a>, an alignment-free algorithm for constructing scaffolding graphs from reads, and <a href="https://github.com/digenoma-lab/FastKM" target="_blank">FastKM</a>, a lightning-fast tool for matching k-mers using rolling and perfect hashing.</p>
    
            <p>In the field of cancer genomics, we have developed a range of tools for detecting somatic mutations and structural variants, including <a href="https://github.com/digenoma-lab/purple-nf" target="_blank">Purple-nf</a>, a <a href="https://www.nextflow.io/" target="_blank">nextflow</a> pipeline for somatic copy number variant calling using PURPLE, and <a href="https://github.com/digenoma-lab/sv_somatic_cns" target="_blank">Sv-somatic-cns</a>, a consensus calling pipeline for somatic structural variants integrating popular SV callers. We also offer tools for detecting mRNA fusions and discovering extrachromosomal DNA in cancer genomes.</p>
    
            <p>Our software is also designed to handle large genomic datasets, such as that generated by high-throughput sequencing machines like MGI-G400 or Nanopore P2-solo. For example, our K-Count pipeline uses k-mer counting to estimate genome size from whole-genome sequencing data, while our Alnsl pipeline provides a fast and accurate alignment tool for short WGS reads.</p>
    
            <p>In summary, our software tools are designed to be scalable, reproducible, and optimized for high-performance computing environments. Whether working with complex genomic datasets or trying to solve complex biological problems, we have the software tools you need to get the job done.</p>
    
            <p>Here are some of the software tools we’ve developed:</p>
    
            <p>Genome Assembly</p>
    
            <p>
                <ol>
                    <li><a href="https://github.com/adigenova/wengan" target="_blank">Wengan</a>: An accurate and ultra-fast hybrid genome assembler that can handle short-read and long-read data from various sequencing platforms. Wengan is designed to be highly scalable and can assemble genomes of various sizes and complexity. Notably, it can assemble a human genome in a single day on a modest machine (20 cores, 40Gb RAM)</li>
                    <li><a href="https://github.com/adigenova/Fast-SG" target="_blank">Fast-SG</a>: An alignment-free algorithm for ultrafast scaffolding graph construction from short or long reads. Fast-SG can be used to generate high-quality genome assemblies in a fraction of the time of traditional assembly methods.</li>
                    <li><a href="https://github.com/digenoma-lab/FastKM" target="_blank">FastKM</a>: A tool for ultra-fast matching of k-mers using rolling and perfect hashing. FastKM is designed to be highly efficient, allowing researchers to generate haplotype-resolved assemblies quickly and accurately.</li>
                    <li><a href="https://github.com/digenoma-lab/hic-scaffolding-nf" target="_blank">hic-scaffolding-nf</a>: A Nextflow pipeline for scaffolding genome assemblies with Hi-C reads. This tool can help researchers improve the contiguity and accuracy of their genome assemblies by leveraging long-range chromatin interaction data.</li>
                    <li><a href="https://github.com/digenoma-lab/k-count-nf" target="_blank">k-count</a>: A Nextflow pipeline to count k-mers and estimate genome size from WGS data. K-count is a fast and efficient tool for genome size estimation, which is essential for many downstream genomic analyses.</li>
                </ol>
            </p>
    
            <p>Cancer Genomics</p>
    
    
            <p>
                <ol>
                    <li><a href="https://github.com/digenoma-lab/purple-nf" target="_blank">purple-nf</a>: A Nextflow pipeline for somatic Copy Number Variant (CNV) calling with PURPLE. This tool can help researchers identify somatic CNVs in tumor samples, providing insights into cancer development and progression.</li>
                    <li><a href="https://github.com/digenoma-lab/sv_somatic_cns" target="_blank">sv_somatic_cns</a>: A tool for consensus calling of somatic structural variants from paired WGS data. This tool can help researchers identify somatic structural variants in cancer genomes, which can provide insights into the mechanisms underlying cancer development or identify complex mutational processes like chromothripsis.</li>
                    <li><a href="https://github.com/digenoma-lab/ampliconarchitect-nf" target="_blank">ampliconarchitect-nf</a>: A Nextflow pipeline to discover extrachromosomal DNA (ecDNA) in cancer genomes. This tool can help researchers identify ecDNA in tumor samples, which has been associated with aggressive tumor growth and poor prognosis.</li>
                    <li><a href="https://github.com/digenoma-lab/nf-gene-fusions" target="_blank">nf-gene-fusions</a>: A Nextflow pipeline to call somatic mRNA fusions. This tool can help researchers identify gene fusions in cancer genomes, which can play a role in tumorigenesis.</li>
                </ol>
            </p>
    
    
            <p>General Utilities</p>
    
    
            <p>
                <ol>
                    <li><a href="https://github.com/digenoma-lab/alnsl" target="_blank">alnsl</a>: A Nextflow pipeline for alignment of short WGS reads. Alnsl is a fast and efficient tool for read alignment, which is an essential step in many genomic analyses.</li>
                    <li><a href="https://github.com/digenoma-lab/longreadstats" target="_blank">longreadstats</a>: is a bioinformatics best-practice analysis pipeline for computing long-read statistics with Nanoplot.</li>
                </ol>
            </p>
    
            <p>We’re constantly updating and improving our software tools to stay at the forefront of genomic analysis. If you’re interested in using any of our software, or if you have ideas for new tools we could develop, don’t hesitate to get in touch!</p>
    
        </div>
    </div>
    
    


    <script src="js/script.js"></script>
    <script src="https://kit.fontawesome.com/41bcea2ae3.js" crossorigin="anonymous"></script>
</body>
{% endblock %}